nest_elephant_tvb/translation/RichEndPoint.py

```diff
@@ -95,8 +87,6 @@ def close_and_finalize(port_send, port_receive, logger_master):
     MPI.Finalize()
```

Comment:
It could be removed since mpi4py states the following: 
`NOTE: MPI_Finalize() is registered (by using Python C/API function Py_AtExit()) for being automatically called when Python processes exit, but only if mpi4py actually initialized MPI. Therefore, there is no need to call MPI.Finalize() from Python to ensure MPI finalization.`
source: https://mpi4py.readthedocs.io/en/stable/overview.html

Reply:
It can be interesting to avoid some call of MPI after the end of the simulation.